### ISSUE TITLE
HW 1 Issue Response - Section 50

### ISSUE THREAD

**Issue by michaelmusick** [Montana-Media-Arts/341-work#1]
Please respond to the open issue and discuss your experiences this week as well as things you expect you might get out of this course. Please feel free to build your comments off of each other. You can specifically respond to other people using the "@" following by there GitHub username.

**Comment by ghost** [comment]
This is a test response to the "HW 1 Issue".

**Comment by tjgambardella1991** [comment]
I signed up for this course with the goal of learning to create, design, and manage my own web page, and after reading the introduction to the course it sounds like that is exactly what we'll be learning. Im looking forward to the weekly lessons and taking in as much as I can.

**Comment by marchendrickson** [comment]
After learning that this course was a purely online course I was definitely a little frightened and nervous about how well I would learn the subject material. However, after taking the first weeks module I am much more confident about what I will get out of this class. I am taking this class as an elective because being able to build a website, or just understand the process that goes into making them is an exceptional skill to have. 
It was very interesting for me to learn how they connect the fiber-optic cables from the sea up onto the landlines, I would've thought it would be a quick just plug and done. I also enjoyed the extra  javascript.com website. I was certainly a little overwhelmed when reading all of the material but taking the 10 minutes to go through the site helped me understand that while its not simple writing code, it is doable for a person like me. I can't wait to keep learning and look forward to what I will be able to achieve at the end of the semester.

**Comment by KaylinKeya** [comment]
I am a Media Arts major and this is my first semester fully indulging in learning about the field and I am very excited. I am very nervous about everything being online as I am more of a visual one-on-one learner, but hopefully if I stay on top of the material and ask for help I'll be A-Okay! I'm very much looking forward to learning everything this course has to offer.

**Comment by ghost** [comment]
I agree with @marchendrickson, it was interesting to learn about how the internet is brought to people around the world. I had no idea that there were literal cables in the ocean! This week's introduction has already gotten me excited to move forward and learn more about web design--and I have a new goal to learn some of these coding languages! I also look forward to learning more about my own computer. I was aware that you could navigate your computer through something like a CLI, but I didn't know it would be simple enough that a total novice like me could play around with it.

**Comment by katecier** [comment]
I watched the videos and read the introduction on the course page.  What really surprised me was that we are able to call up a directory simply by typing things and not using the mouse at all.  This made me think of the technology and all of the different codes that are written so users can complete a simple task on a computer like opening a folder.  Like @marchendrickson  said, it was surprising to learn that the internet runs because of cables that run across the ocean.  I had always thought of the internet as the cloud, and nothing you could physically touch. I also learned that HTML was a language that is used to create web pages.  I had learned about Tim Berners Lee in the past and his invention of the web or HTTP but now know how the information he wrote is put into play and how it organizes web pages and its job to connect someone to anyone else on the server. 

I am looking forward to learn some basic coding and excited to create and personalize my own website while knowing all the hidden codes in play that make things appear simple on the screen.

**Comment by ZachKrell** [comment]
Oh man! I still feel like I'm in a little over my head, I tend to be very clueless when it comes to stuff like this but I push through. However, this first week worth of information has made me feel so much better. I still have a lot of questions and am confused by a lot, but I feel like with the weekly "Modules" I should be able to keep up nicely.  I'm hoping that through this class I will be able to build and maintain a personal website.  This week also made me realize how awesome Fiber Optics is. My parents live in Austin, and I am so jealous that they will have the chance to get fiber optic technology while I'm here stuck with Charter spectrum.

**Comment by gordonty** [comment]
Like @ZachKrell I fell like I'm in a little over my head. I'm hoping that as we work through the course that feeling will go away. I found it interesting that http and html aren't just letters put together but that they actually mean something. Kind of like how IDK means I don't know. I knew the internet was a little complicated but I didn't realize how complicated it really is. It's like the human body, complicated but amazing.

**Comment by StephenRound** [comment]
After watching the videos and playing around with the powershell, I feel a little more confident. One technique I'm going to deploy to help me memorize the shorthand commands (and you're all welcome to use this too) is to have a piece of tape on my laptop with the shorthand commands written on them. Hopefully before too long I won't have to look at it constantly while using the powershell. 

It was interesting to learn how the Internet works as well. I always imagined it as "internet waves" just wafting through the air and finding their way onto your phone/computer/tablet etc. Knowing how the Internet works proved a lot more intriguing than thinking how it works. 

Hopefully, with the tiny bit of coding experience I have and a good sense of time management I can stay ahead of things in this class.

**Comment by misschinatown** [comment]
I am new to online courses and initially was overwhelmed with course work, but I find the videos to be of great help! I spent plenty of time on the Code School Web Development because I actually enjoyed it. 

I have designed a few web pages using ready-to-use website building platforms. I am very excited to learning the process of development!

**Comment by DaveCowan636** [comment]
I was slightly amazed at how much cable has been laid for the internet.  Always thought anymore it was more to do with satellite.   Ive had some experience with HTML code in computer modeling, but didn't realize that website design would be similar.  Its going to be an interesting class, I'm looking forward to it.

**Comment by HaleyTat** [comment]
It was so interesting to discover that 10-15 different routers are used to obtain information for just one personal laptop. The behind the scenes knowledge of the internet just shows how intricate and physical the process of the internet. It turns this almost intangible object, to become more and more concrete.

**Comment by jonchance13** [comment]
I found the introductory readings and videos very helpful, and yet at the same time a bit intimidating. There seems to be no short supply of depth behind the electronic veil. This course will be of great benefit to us all!

**Comment by ashtonmwilson** [comment]
Your not alone @ZachKrell, @gordonty, I am a little overwhelmed, and super excited to be joining y'all for this course. I have worked a bit on the layout and design side of websites, and have not spent any time on the coding side. Many of the videos this week were good refreshers on the history and structure, And I need to work on memorizing
<img width="523" alt="screen shot 2017-01-27 at 6 08 44 pm" src="https://cloud.githubusercontent.com/assets/14146228/22398418/719dd74c-e545-11e6-95d5-87bb5de02fdc.png">
 my TLA's (three letter acronyms) I appreciate the format and have been taking screen shot notes (Cmd. shft 4 on mac) of key concepts I want to remember later.

for those using Unix, I found this helpful: https://kb.iu.edu/d/abdb

**Comment by rebekahH** [comment]
I'm very interested by the way this course is laid out. I feel like I am in over my head and am very overwhelmed, if I am to be honest. In the Andrew Blum video he uses a quote from Neal Stephenson; "Wired people should know something about wires..." and that is what I feel that this class is going to teach me, the magic that happens behind the curtain or a peak under the hood.

**Comment by ARoomFullofBees** [comment]
I'm very intimidated by this course already as well.  Though, I'm certainly much clearer on how the internet works after watching the videos. 

I also have to admit I'm a little glad so many other people here share my apprehension. Hopefully, we'll be able to tackle this subject matter together and come out with some very useful new skills.

**Comment by laurasugi** [comment]
I have always imagined the internet to be a very abstract concept and given it very little thought despite it being something I use daily. Seeing how it actually works, and the actual mechanics and physical nature of it were very interesting to me. Similarly, I enjoyed the tutorial on Powershell and learning to better control a machine that I have used daily for years without more than a basic understanding of. 

As someone who is constantly using the internet and computers for work as well as personal reasons; I am excited to learn more about not just how to work with them but how they actually function.  Specifically, I am most looking forward to learning more about code and building a website.

**Comment by MistaSierra** [comment]
I think there was a bit too much information thrown in all at once. But hey the good thing about online classes is that you can go back and re-read/watch everything, which i'm positive I will have to do multiple times. What I found interesting was many different types of ways and tools available for making a website. Just the thought of learning one way is daunting.

**Comment by shelbsjean** [comment]
I took this course in hopes to better understand the digital world we live in. I have already learned so much about how the internet works and what goes into computer commands that I take for granted. I never considered that there are physical wires that run on the ocean floor to connect us around the world through the internet. I am very thankful that the instructors have provided supplement readings and videos that will greatly help me because I am a beginner in all aspects of these concepts. So far all the information and material is very interesting to me and I am excited to learn more about web development.

**Comment by darbymcnally** [comment]
After doing the Week1 course load I am intrigued and excited! The Internet now engulfs the world we live in, so it seems rather foolish to not be familiarized with how it works. Just as many others stated above, I had no idea that there were actual wires stretching across the oceans. I am also excited to work on my terminology more. There are several familiar abbreviations I have heard, but never have I taken the time to figure out what they mean. I haven't really taken an online class before, so I think time-management is going to be a factor for me in order to not wait the last minute.

**Comment by aspen-runkel** [comment]
After taking in all of the material presented to us this week during our introduction to the course, I found that most of the information was either new to me or my current knowledge on the subject was limited.  I am a Business Marketing student in my third year also working on a Digital Marketing Certificate and Media Arts Minor, and I choose this class as one of my electives for the minor.  Based on the overview and expectations of the course, I am happy I chose this course as one of my electives as it appears that it will be useful to my future in the business industry.

As many others have mentioned, I found the “Internet” section very interesting and eye-opening.  It certainly helped me see the internet as a physical entity rather than this abstract object that it is often perceived as.  I had heard many of the terms associated with the internet before (including fiberoptics, servers, packets, IP addresses, routers, bandwidth, bitrate, etc.), but this section helped me better understand how they all work and function together.

In terms of the “Web Development” section, the “Beginners Guide to Web Development” at codeschool.com was an excellent resource that I will need to spend more time with to feel comfortable.  This is where I started to feel a little out of my element, and the final section on “Your Development Environment” was completely new to me.  I’m relieved to hear that others feel a bit overwhelmed when it comes to command-line interface (CLI) and performing actions within them, but I am optimistic.

**Comment by hyeink** [comment]
Before I actually learned all of the materials of this week, I thought I knew pretty much everything about Internet and how it works. I thought I can make my own website easily through wordpress or those kind of websites that are already made it for me. I was more interested after I went through all of videos and readings for this week. I didn't know the fact that personal computer like home computer is not directly connected to the Internet. I figured out that I didn't have any questions of it while I was using Internet before. I'm excited to learn about more so when I use Internet or make website I can question about how it is made and how it works. Also, I knew about the CLI, but I didn't know specific things about it. I didn't know that it can speed up the development process and display more information than is typical of a traditional GUI based environment. I'm excited of rest of the semester to learn this course!

**Comment by estewart810** [comment]
After going through this first weeks course load, Im feeling both terrified and excited. Web design is important to my major (media art), but I feel like Im the last person on earth who would do well in web design. But at the same time I had a lot of fun learning about the internet. I already knew how the internet was started but I didn't really know how it worked. I learned about all the the different ways the signal can be sent , and I also had know idea that the internet is sent in giant cables across the ocean. That is absolutely amazing to me. Learning about the internet really gave me a new appreciation for it and Im very pumped for this semester!

**Comment by mason-birgenheier** [comment]
I am very new to Web Design and I feel that it this course is suitable for everyone, especially artists, entrepreneurs, etc. because of the importance and prominence the internet brings to the world we live in today. I am excited to learn as much as in this course and have a better understanding of what it means to design and manage a website.

**Comment by danieljhill1990** [comment]
I am pretty new to web-based coding, and design. The only real experience I have is from my Myspace days of long ago. My wife is a Graphic Designer, so seeing what she does really peaked my interest in the field. I hope to learn a lot. As a distance education guy (based in TX, from LA), this course should be loads of fun.

**Comment by egriffin2** [comment]
@ZachKrell agreed!  I am also feeling a little overwhelmed right now!  After having taken Creative Coding I last semester as a live class, it feels really weird and intimidating to be learning this all online. I am very excited to get a grasp on CSS and HTML over the course of the semester, and I'm looking forward to getting started on class work!

**Comment by tannercurey** [comment]
I found the videos in Week 1 to be intriguing and informative. It's amazing how much we overlook how the tools we use everyday actually function. My web design experience is fairly limited to Wordpress and that experience has been using fairly basic functions for personal blogs. I'm excited to learn more about the functions of the internet and how to actually operate my own site. There was so much discussed in the videos that I had no prior knowledge of. I was unfamiliar with CLI (Command-line Interface) and PowerShell and my understanding of HTTP and HTML is certainly limited. Needless to say I have a lot to learn and i'm looking forward gaining a better understanding of the realm of web design.

**Comment by JakeTheFlare** [comment]
I have taken a few web design classes in the past, and I have a bit of a standard knowledge about HTML coding and how to structure a website, though I am hoping to further improve my skills by taking this course.  I also learned about how the internet works in my Digital Technology class last semester, and based on what I learned from that and watching the Week 1 videos, I find it very interesting.  Pretty cool how the internet actually shares information with a whole bunch of other networks and servers in order to deliver it to the destination.  I am starting to get the hang of using Terminal on my computer and the directories, though I still have a lot of things I hope to learn and need to practice on, like; how to open existing files and programs, moving files, and all that stuff.  All in all, this may be a challenging class, but I believe I can get through it all if I just follow directions and ask for help when I need it.

**Comment by lrjlavalley222** [comment]
I am excited mostly to become more in tune with the computer i use everyday. It is so easy in this day and age to just get tablets, phones, computers, etc. and not understand how they work or how to have full control over them. I guess that is why I can't wait to dive in and nerd out on new information.

**Comment by shanellerparker** [comment]
I initially registered for the class so that I could create better websites for my own personal use because I felt limited using sites such as Wix or Wordpress. Now, I feel pretty intimidated at the thought of this challenge because of the information that the videos and guide that was assigned this week. However, I am also excited about the knowledge that I gained this week about the internet, coding, and manipulating one's own computer. On a side note I also found it humorous that some of the most abstract ideas about the internet being a cloud or something above were exactly the opposite of were the source is located.

**Comment by ashlynlyons13** [comment]
I joined this class wanting to add basic web design to my resume to go along with a communications degree focusing in public relations and advertisment. You can never have too much tech experience this day in age. But as many have already said, I originally felt ovrwhelmed. After going through everything, I can see there is a lot to learn but feel better after feeling through the format of this class and watching the videos that go along with it. I have basic computer knowledge and have designed sites on prexisting templates like wordpress, so I am eager to learn the coding side and be able to create a page entirely on my own.

**Comment by ashlynlyons13** [comment]
@DaveCowan636 I also pictured more of a 
satellite approach whenever I questioned how exactly the internet functioned. To find out there is physical cables running through the ocean is very interesting. Different areas all over the world receive their "connection" in different ways. I guess because wifi and 3G are so prevalent in our lives that we don't  always think about how technology may work behind the scenes in other places.

**Comment by CRDecker96** [comment]
The best thing I could get from the class is knowing and understanding how to run and operate other coding like html and Css, I have past coding knowledge with Dr.Musick and he referred me to checking the class out. The challenge of learning a new way to code is always fun once you understand the process. I enjoy the building and creation aspect of web development and how creative it can be, but the hardest part I have is keeping files and work organized, thats half the battle for me. I don't think this class is like a "how to work Wix".  I hope its more of the understanding of web building through processing of code, like creating an old Myspace page but you write the code.

**Comment by jackson-wagner** [comment]
I don't really have any experience with web design, but I'm going into journalism and feel like it will be incredibly helpful if I am able to code and work on websites. I'm an online editor right now and control a lot of what goes on on our website, so I think this class will help me improve at my job. Like a lot of people, I'm also a little overwhelmed with the start, but I'm excited to see where this class goes.

**Comment by mederioswhitworth-babb** [comment]
@jackson-wagner I also think that this would be a good skill to have as a journalism major. Like a lot of you I do not have really any experience with the web but I am hoping that this class will help with that. I felt overwhelmed with the different things that were thrown at us for week one, especially because I just added this class yesterday. I hope that I can catch up and still take in as much information as I can. I thought that the idea of programming is really interesting and a skill that I think will be difficult but really useful no matter what kind of job I get in the future.
